src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -103,6 +106,18 @@ class IqPolicyEvaluatorUtil
     ExceptionUtils.indexOfType(throwable, IOException) >= 0
   }
 
+  private static Properties getAdvancedProperties(final String inputPropertiesString, final LoggerBridge loggerBridge) {
+    Properties advanced = new Properties()
+    if (StringUtils.isNotEmpty(inputPropertiesString)) {
+      try {
+        new StringReader(inputPropertiesString).withCloseable {advanced.load(it)}
+      } catch (IOException e) {
+        loggerBridge.error("Unable to parse advanced properties: ${e.message}", e)
+      }
+    }
+    advanced
```

Comment:
Trivial: I don't think its on our 'opinionated preferences' page, but the team has discussed in the past and prefers an explicit `return` on return values over the groovy voodoo of last statement gets returned.